### PR TITLE
Updated  the login article 

### PR DESCRIPTION
**Pull request by jonat123**
Updated the greeting in the login article with the correct information.

### CODE REVIEW

File: umbraco-cms/fundamentals/backoffice/login.md [hunk 1]
```diff
@@ -88,7 +94,7 @@ The `LoginBackgroundImage` and `LoginLogoImage` are referenced from the `/wwwroo
 
 To affect the styling of the images add a folder to the `/App_Plugins` folder containing a `CSS` folder, containing a CSS file, and a package.manifest file. An example structure might be:
 
-```
+```package.manifest
```
Comment (sofietoft) [inline umbraco-cms/fundamentals/backoffice/login.md:97]: Not sure if "package.manifest" is a syntax? 🤔

File: umbraco-cms/fundamentals/backoffice/login.md [hunk 2]
```diff
@@ -13,13 +13,16 @@ You will be presented with a login form similar to this:
...
+Below, you will find instructions on how to customize the login screen.
 
 ## Greeting
 
-The login screen features a greeting which you can personalize by overriding the existing language translation keys. To do this, create a 'user' translation file for the default language of your Umbraco site, (usually en-US) to override the greetings. For en-US, you'd create a file called: `en_us.user.xml` in the directory `~/config/lang/`. Then take the relevant keys (listed below) and add them to your `~/config/lang/en_us.user.xml` file, and update the greetings as necessary!
+The login screen features a greeting that you can personalize by overriding the existing language translation keys. To do this, create a 'user' translation file for the default language of your Umbraco site, (usually en-US) to override the greetings. For en-US, you'd first need to create a ```config``` folder at the root of your project.
+
+In the config folder create a folder called ```lang``` and create a file called: `en_us.user.xml`. Then take the relevant keys (listed below) and add them to your `~/config/lang/en_us.user.xml` file, and update the greetings as necessary!
```
Comment (sofietoft) [inline umbraco-cms/fundamentals/backoffice/login.md:22]:
```suggestion
In the config folder create a folder called `lang` and create a file called: `en_us.user.xml`. Then take the relevant keys (listed below) and add them to your `~/config/lang/en_us.user.xml` file, and update the greetings as necessary.
```

**Review by sofietoft — COMMENTED** [review]
LGTM 💪 
Just added a couple of suggestions.

Also, something I couldn't add as a suggestion:
Lines 26-34 should be indented, as it is inside the `language` tag now.

File: umbraco-cms/fundamentals/backoffice/login.md (hunk 1 above)
Comment (jonat123) [reply umbraco-cms/fundamentals/backoffice/login.md]: No its not,

just read our docs and it should be JSON, so i have changed it :)

**Review by sofietoft — APPROVED** [review]
Looks good 💪

**Pull request merged**